doc/swagger.v1.yml

```diff
@@ -106,6 +184,47 @@ definitions:
         description: language code
     required: ['id', 'name', 'email']
 
+  FinancialTransaction:
+    type: object
+    properties:
+      id:
+        type: integer
+      user_id:
+        type: ['integer', 'null']
+        description: id of user who entered the transaction (may be <tt>null</tt> for deleted users or 0 for a system user)
+      user_name:
+        type: ['string', 'null']
+        description: name of user who entered the transaction (may be <tt>null</tt> or empty string for deleted users or system users)
+      amount:
+        type: number
+        description: amount credited (negative for a debit transaction)
+      note:
+        type: string
+        description: note entered with the transaction
+      created_on:
```

Comment:
why is there `created_on` and not `created_at`?

Reply:
Oops, outdated doc - thanks!
Too bad this wasn't caught by apivore :/ (since it was present in `required`).